ui/urls.py

```diff
@@ -11,6 +12,7 @@
     url(r'^$', views.Index.as_view(), name='index'),
     url(r'^register/$', views.register, name='register'),
     url(r'^login/$', views.ui_login, name='login'),
+    url(r'^logout/$', django_logout_view, {'next_page': '/login/'}),
```

Comment:
If `/login/` shouldn't be used with touchstone I think `next_page` should be something else, maybe just `/`?